phone-store/angular.json

```diff
@@ -106,19 +106,19 @@
             "tsConfig": [
               "tsconfig.app.json",
               "tsconfig.spec.json",
-              "e2e/tsconfig.json"
+              "cypress/tsconfig.json"
             ],
             "exclude": [
               "**/node_modules/**"
             ]
           }
         },
         "e2e": {
```

Comment:
You can change this to `cypress` if you would prefer the command to be `ng cypress` I think.

Reply:
We tried that, and it does not seem to be an option.

Reply:
Oh ok my bad. I thought it was more flexible than that but I never actually tried it. You're right it can only be a select set of keys.